Assets/Script/GameManager.cs

```diff
@@ -1,25 +1,25 @@
-﻿using System.Collections;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 using System;
 
 public class GameManager : MonoBehaviour
 {
-    public Text PlayTime;
+    [SerializeField]
+    private Text PlayTime;
+    
     private TimeSpan startTime = new TimeSpan(0, 10, 0);
-private TimeSpan remainingTime;
-
-    private void Start()
+    public TimeSpan remainingTime { get; set; }
+    
+    private void OnEnable()
     {
```

Comment:
Initialize remainingTime
```suggestion
    {
        remainingTime = startTime;
```